airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -1264,6 +1265,14 @@ def _run_scheduler_loop(self) -> None:
                     except Exception:
                         self.log.exception("Something went wrong when trying to save task event logs.")
 
+                with create_session() as session:
+                    for deadline in session.scalars(
+                        select(Deadline)
+                        .where(Deadline.deadline_time < datetime.now(timezone.utc))
+                        .where(Deadline.callback_state.is_(None))
```

Comment:
Maybe add a brief comment about how the `callback_state` being `None` implies that the deadline is unhandled

Reply:
I can do that